src/handlers/use_handler.rs

```diff
@@ -302,7 +302,7 @@ async fn copy_file_with_error_handling(old_path: &Path, new_path: &Path) -> Resu
 /// add_to_path(&installation_dir).unwrap();
 /// ```
 fn add_to_path(installation_dir: &Path) -> Result<()> {
-    let installation_dir = installation_dir.to_str().unwrap();
+    let installation_dir = installation_dir.to_str().unwrap().replace('/', "\\");
```

Comment:
Why here do you replace the front-slash with a back-slash despite this part also affecting unix?

Reply:
Oh right, only in the new PR does it actually affect unix, move this inside the windows if statement so i dont have to deal with merge conflicts.